snapcast-server/snapserver.conf

```diff
@@ -0,0 +1,106 @@
+###############################################################################
+#     ______                                                                  #
+#    / _____)                                                                 #
+#   ( (____   ____   _____  ____    ___  _____   ____  _   _  _____   ____    #
+#    \____ \ |  _ \ (____ ||  _ \  /___)| ___ | / ___)| | | || ___ | / ___)   #
+#    _____) )| | | |/ ___ || |_| ||___ || ____|| |     \ V / | ____|| |       #
+#   (______/ |_| |_|\_____||  __/ (___/ |_____)|_|      \_/  |_____)|_|       #
+#                          |_|                                                #
+#                                                                             #
+#  Snapserver config file                                                     #
+#                                                                             #
+###############################################################################
+
+# default values are commented
```

Comment:
I always find this to be a difficult call, but have you considered cutting this file down to just the lines we're actually using? I think it makes it a lot clearer to see what's going on. Looks like it might be able to be cut down to simply:
```
[stream]
stream = pipe:///var/cache/snapcast/snapfifo?name=default
sampleformat = 44100:16:2
```

Reply:
Agree with this being a difficult call. My biggest reason not to cut it down is to keep the options visible but its also rarely used, since you would be checking documentation if you were to change something. +1 to cutting it down